README.md

```diff
@@ -37,6 +37,8 @@
 
  This is a mobile web application where user can manage their budget: user have a list of transactions associated with a category, so that they can see how much money they spent and on what.
 
+- [Click here](https://www.loom.com/share/ed1a05f7ea4d4b5c911e95e9d7fab2c9) to watch the overview video
```

Comment:
- [ ] Great job with the video presentation explaining the features of the app and also talking about what you are proud of making, however, you did not highlight the piece of code you are proud of or finds interesting as we can see in the project requirements
![Screenshot from 2023-06-09 04-05-52](https://github.com/hienphan0111/budget-app/assets/46388113/8c4c5af4-33a4-41ac-a9ed-0280efab87ad)

Reply:
This change has been done well